src/Microsoft.DotNet.ProjectModel/AnalyzerOptions.cs

```diff
@@ -15,7 +16,7 @@ public class AnalyzerOptions
 
         public static bool operator ==(AnalyzerOptions left, AnalyzerOptions right)
         {
-            return left.LanguageId == right.LanguageId;
```

Comment:
Well, I'll nit pick and say that I think your way produces an unnecessary vcall, but I don't really care either way :)